js/Common.js

```diff
@@ -1100,6 +1100,20 @@ if (!CRM.vars) CRM.vars = {};
         title: title,
         type: type
       };
+      switch(type) {
+        case 'success':
+          params.icon = 'fa-check';
+          break;
+        case 'info':
+          params.icon = 'fa-info';
+          break;
+        case 'error':
+          params.icon = 'fa-times';
+          break;
+        case 'alert':
+        default:
```

Comment:
This is the checkstyle failure, which seems like more of a failure of imagination on the part of the linter.  If it's important to have separate cases, I can obviously change this.

Reply:
If the case is the same as the default, you don't need it :)

Reply:
Oh alright...